IdentityCore/src/telemetry/request_telemetry/MSIDLastRequestTelemetry.m

```diff
@@ -40,6 +64,8 @@ @interface MSIDLastRequestTelemetry()
 
 @implementation MSIDLastRequestTelemetry
 
+static bool shouldReadFromDisk = YES;
```

Comment:
just curious, why would we ever not read from disk? And should it be an initializer parameter and not a static boolean?

Reply:
Since the main use case we have in mind is when a user kills their app, the idea was that we should read from disk only the first time the lastRequestTelemetry object is created, so we can retrieve saved data and then continue normal use. This way, we don't need to do extra reads from disk as well.